### PR TITLE
Complete helloWorld and votingEligibility (question 1, 2)

### PR DESCRIPTION
**Pull request by ManoolK**

### CODE REVIEW

File: force-app/main/default/classes/FlowControlLoopsCollections.cls [hunk 1]
```diff
@@ -42,7 +45,7 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        return age >= 18;
```
Comment (ManoolK) [inline force-app/main/default/classes/FlowControlLoopsCollections.cls:48]: Test comment 1

File: force-app/main/default/classes/FlowControlLoopsCollections.cls [hunk 2]
```diff
@@ -29,7 +29,10 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        if (x == null || y == null || x <= y) {
+            return null;
+        }
+        return 'Hello World!';
```
Comment (chase-cca) [inline force-app/main/default/classes/FlowControlLoopsCollections.cls:35]: This is the optimal way of doing this problem, I'd say. Great thinking.

File: force-app/main/default/classes/FlowControlLoopsCollections.cls (hunk 1 above)
Comment (chase-cca) [inline force-app/main/default/classes/FlowControlLoopsCollections.cls:48]: Excellent.

**Review by chase-cca — COMMENTED** [review]
All set.

**Pull request merged**